src/CommunityToolkit.Maui/Views/Popup/PopupOptions.shared.cs

```diff
@@ -0,0 +1,128 @@
+﻿using Microsoft.Maui.Controls.Shapes;
+
+namespace CommunityToolkit.Maui;
+
+/// <summary>
+/// Popup options.
+/// </summary>
+public partial class PopupOptions : BindableObject, IPopupOptions
```

Comment:
Why this is a bindable Object? Took a look into our samples and didn't find any bindable usage for this type... IMO this doesn't need to be Bindable at all

Reply:
You're correct that we should have a sample demonstrating bindings. 

Yes, it needs to be a BindableObject. Using BindableProperties is the only way for a developer to dynamically change the values of PopupOptions while the Popup is being displayed.